Week1/homework/js-exercises/logArray.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+{
+  let myClothes = [];
+  console.log('pants', 'shirt', 'shoes');
+  console.log(myClothes);
+  let myArrAnimal = ['Giraffe', 'Elephant', 'Lion'];
+  console.log(myArrAnimal);
+  myArrAnimal.push('Piglet');
+  console.log(myArrAnimal);
+}
```

Comment:
👍 for using `use strict` and using curly braces to encapsulate your code. However, I think using `const` instead of `let` for creating those variables is a good idea because we are not reassigning those variables anywhere, right?